source/BrainPadDisplay.cpp

```diff
@@ -0,0 +1,56 @@
+#include "BrainPadDisplay.h"
+
+using namespace codal;
+
+//static BrainPadDisplay *device_instance = NULL;
+
+BrainPadDisplay::BrainPadDisplay(Pin& sda, Pin& scl): i2c(sda, scl) {
+  
+	//device_instance = this;
+
+}
+
+void BrainPadDisplay::WriteCommand(int cmd) {
+	Data[1] = (byte)cmd;
+	i2c.write(DeviceAddress, Data, 2, 0);
+}
+
+void BrainPadDisplay::InitScreen() {
+
+	WriteCommand(0xae);//--turn off oled panel
```

Comment:
What do the varying amounts of dashes signify?

Reply:
Gus asked me to copy the code from page 19 here, I hadn't noticed the difference in dashes until you pointed out. Not sure what they mean, probably something relating to the datasheet. Should I just make them all uniform? 

https://cdn-shop.adafruit.com/datasheets/UG-2864HSWEG01+user+guide.pdf

Reply:
We should verify the license/copyright before just copying code from some external source